src/Nri/Ui/UiIcon/V1.elm

```diff
@@ -2118,3 +2122,297 @@ sync =
+                        ]
+                        []
+                    ]
+                ]
+            ]
+        ]
+        |> Nri.Ui.Svg.V1.fromHtml
+
+
+{-| -}
+graduateCap : Nri.Ui.Svg.V1.Svg
+graduateCap =
+    Svg.svg
+        [ Attributes.width "100%"
+        , Attributes.height "100%"
+        , Attributes.fill "currentcolor"
+        , Attributes.viewBox "0 0 101 65"
+        , Attributes.version "1.1"
+        ]
+        [ Svg.g
+            [ Attributes.stroke "none"
+            , Attributes.strokeWidth "1"
+            , Attributes.fill "none"
+            , Attributes.fillRule "evenodd"
+            ]
+            [ Svg.g
+                [ Attributes.transform "translate(0.984790, 0.051995)"
+                , Attributes.fill "#F3336C"
```

Comment:
Is _here_ where the `currentcolor` should go instead? Currently the icon is noredink-red:
<img width="68" alt="" src="https://user-images.githubusercontent.com/8811312/178030913-60c2b206-befa-4eca-997a-12aeee3750a7.png">